ERCS/erc-7769.md

```diff
@@ -0,0 +1,546 @@
+---
+eip: 7769
+title: JSON-RPC API for ERC-4337
+description: JSON-RPC API methods for communication between smart contract account wallets and ERC-4337 bundlers
+author: Vitalik Buterin (@vbuterin), Yoav Weiss (@yoavw), Dror Tirosh (@drortirosh), Shahaf Nacson (@shahafn), Alex Forshtat (@forshtat)
+discussions-to: https://ethereum-magicians.org/t/erc-7769-json-rpc-for-erc-4337-account-abstraction/21126
+status: Draft
+type: Standards Track
+category: ERC
+created: 2024-08-23
+requires: 4337, 7562
+---
+
+## Abstract
+
+Defines new JSON-RPC API methods which enable [ERC-4337](./eip-4337) wallets to communicate with `UserOpeation` mempool
+nodes and bundlers, matching the functionality that exists for Ethereum transactions.
+
+Additionally, a set of `debug` JSON-RPC API methods is defined in order to facilitate development, testing and
+debugging issues with ERC-4337 implementations.
+
+## Motivation
+
+In ERC-4337, user transactions as defined in Ethereum are replaced with `UserOperation` objects, which contain all the
+information needed to perform the operations requested by the users.
+
+However, existing Ethereum JSON-RPC API methods are not suited to working with `UserOperation` objects.
+In order to facilitate the operation of the alternative `UserOperation` mempool it is important that all
+implementations of the ERC-4337 protocol have a standardized set of APIs that can be used interchangeably.
+
+## Specification
+
+### Definitions
+
+* **bundler**: a node exposing the APIs, in order to submit them to the network.
+  A bundler collects one or more UserOperations into a bundle and submits them together to
+  the `EntryPoint` in a single `handleOps` call.
+
+### RPC methods (eth namespace)
+
+#### eth_sendUserOperation
```

Comment:
```suggestion
#### `eth_sendUserOperation`
```

Please put code in backticks, even in headings. This applies for the rest of the document as well.